src/Utility/Text.php

```diff
@@ -852,4 +859,75 @@ public static function parseFileSize($size, $default = false)
         }
         throw new InvalidArgumentException('No unit type.');
     }
+
+    /**
+     * Get/set default transliterator identifer string.
```

Comment:
I am not sure if we should introduce more combined setters/getters as we are removing them in future versions because they are an anti-pattern that increases code complexity and error rate.
Wouldnt it make sense to explicitly introduce a getter and setter as separate and distinct methods to avoid more work in the very near future?